10377.xml

```diff
@@ -0,0 +1,247 @@
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource defines the period to run this data sampling. Data Sampling will stop when the Run Period has elapsed from the start time. Depending on the Data Sampling Start Mask's "don't care" field, sampling may re-start again.]]>
+				</Description>
+			</Item>		
+			<Item ID="7">
+				<Name>Reference A</Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Mandatory</Mandatory>
+				<Type>Opaque</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource contains the value of reference data that is to be compared with Data resource in conjunction with Comparison A resource. The comparison result is flagged by Status resource. When Status resource is made observable, this effectively becomes a mechanism to signal the server on a hit. The reference data and data type is formatted as CBOR payload.]]>
+				</Description>
+			</Item>
+			<Item ID="8">
+				<Name>Comparison A </Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Mandatory</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource represents the logical statement to compare Data resource against Reference A resource. The statements are encoded as following:
+				0 = No Comparison
+				1 = Data > Reference A
+				2 = Data >= Reference A
+				3 = Data < Reference A
+				4 = Data <= Reference A
+				5 = Data equals Reference A
+				6 = Data not equal Reference A]]>
+				</Description>
+			</Item>
+			<Item ID="9">
+				<Name>Reference B</Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Opaque</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource contains the value of reference data that is to be compared with Data resource in conjunction with Comparison B resource. The comparison result is flagged by Status resource. When Status resource is made observable, this effectively becomes a mechanism to signal the server on a hit. The reference data and data type is formatted as CBOR payload.]]>
+				</Description>
+			</Item>
+			<Item ID="10">
+				<Name>Comparison B </Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[This resource represents the logical statement to compare Data resource against Reference B resource. The statements are encoded as following:
+				0 = No Comparison
```

Comment:
I would recommend to format all these fields as `<value>: <description` to follow the style of IPSO objects. That makes it possible to parse these programmatically.

```suggestion
0: No Comparison
```

Reply:
changed to ':'